install/installer/pkg/config/v1/config.go

```diff
@@ -99,6 +100,8 @@ type Config struct {
 
 type Metadata struct {
 	Region string `json:"region" validate:"required"`
+	// InstallationShortname establishes the "identity" of the (application) cluster.
+	InstallationShortname string `json:"shortname,omitempty"`
```

Comment:
```suggestion
	InstallationShortname *string `json:"shortname,omitempty"`
```

May be worth replacing this with a pointer so the `omitempty` is respected? If you do this, the value will default to `nil` rather than an empty string.

If you want to stay with this, can you run `go run . init > example-config.yaml` to generate the new example config please?

Reply:
Thx for the hint, will do!